nexus/src/external_api/http_entrypoints.rs

```diff
@@ -2550,6 +2550,8 @@ async fn image_delete(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// Promote a project image
+///
 /// Promote a project image to be visible to all projects in the silo
```

Comment:
It looks weird when the paragraph copy starts exactly the same as the title. Maybe we could change this to `Make a project image visible to all projects in the silo`

Reply:
I suppose so — but there's a few like that, if we change this we should do the others.